plugins/modules/vmware_cluster_drs.py

```diff
@@ -58,6 +58,8 @@
     drs_vmotion_rate:
       description:
       - Threshold for generated ClusterRecommendations.
+      - 1: lowest
+      - 5: highest
```

Comment:
```suggestion
      - Threshold for generated ClusterRecommendations ranging from 1 (lowest) to 5 (highest).
```
This seems to be interpreted in a very weird way (as a dict `{1: 'lowest'}` and not as a string `"1: lowest"`) which leads to an [error](https://ansible.softwarefactory-project.io/zuul/build/77964af55d00498895ed5e8850403828). It should work if you do it another way, maybe like I am a suggesting.